src/components/MessageBox/messageBox.vue

```diff
@@ -49,7 +49,7 @@
 
 <script lang="ts" setup>
 import BooleanTable from '@/DialogBox/BooleanTable.vue'
-import { defineProps, defineEmits } from 'vue'
+// import { defineProps, defineEmits } from 'vue'
```

Comment:
Please remove instead of commenting lines

Reply:
Yes @tachyons I have removed extra comments from the pr